### ISSUE TITLE
Use Log package

### ISSUE THREAD

**Issue by ScopiCisco** [gherlein/gonetmon#2]
You could use https://golang.org/pkg/log/ so that you don't need to have if(debug)

Just setFlags at the beginning if debug flag is set.

**Comment by dcwangmit01** [comment]
Try this out: https://github.com/sirupsen/logrus

**Comment by gherlein** [comment]
yes, this is the right way to do it.  Thanks for the pointer to logrus Dave!
